portafly/doc/ux_specs/global_components/modal.md

```diff
@@ -26,6 +27,18 @@
     * Submit button [Disabled state until all conditions required to submit the form are met]
     * Cancel button [Always enabled]
       * Dismisses modal
+  * When user submits the form:
+    * the Submit button is replaced by a spinner
+      * spinner should stay visible for at least 1.5 seconds (even when the request is resolved earlier)
```

Comment:
I would suggest deferring this for later, when we implement the API requests properly.
@didierofrivia

Reply:
sounds good to me, but how to make sure we don't forget about this?

Reply:
Maybe Jira issue?